fcrepo-persistence-ocfl/src/test/java/org/fcrepo/persistence/ocfl/impl/DbFedoraToOcflObjectIndexTest.java

```diff
@@ -209,13 +189,11 @@ public void identifierTooLong() throws InvalidResourceIdentifierException {
         for (cnt = 0; cnt < 149; cnt++) {
             root = root + "/" + String.valueOf(cnt);
         }
-        try {
-            final FedoraId tempid = FedoraId.create(root);
+        final var finalRoot = root;
```

Comment:
Wondering if this was necessary?

Reply:
it was needed because `root` needs to be modifiable in the block above, but the value has to be final in order for it to be accessible in the closure of the assertThrows